php_phongo.c

```diff
@@ -522,6 +522,56 @@ mongoc_bulk_operation_t *phongo_bulkwrite_init(zend_bool ordered) { /* {{{ */
 	return mongoc_bulk_operation_new(ordered);
 } /* }}} */
 
+static void phongo_bulk_write_error_add_message(char **tmp_msg, bson_t *errors)
+{
+	bson_iter_t iter;
+
+	bson_iter_init(&iter, errors);
+
+	while (bson_iter_next(&iter)) {
+		bson_t cbson;
+		uint32_t len;
+		const uint8_t *data;
+		bson_iter_t inner_iter;
+
+		if (!BSON_ITER_HOLDS_DOCUMENT(&iter)) {
+			continue;
+		}
+
+		bson_iter_document(&iter, &len, &data);
+
+		if (!bson_init_static(&cbson, data, len)) {
+			continue;
+		}
+
+		if (bson_iter_init_find(&inner_iter, &cbson, "errmsg") && BSON_ITER_HOLDS_UTF8(&inner_iter)) {
+			char *tmp_errmsg = bson_iter_dup_utf8(&inner_iter, NULL);
```

Comment:
Any reason we can't simply use `bson_iter_utf8()` here, since `tmp_errmsg` is not going to be modified and we're just going to copy it into `tmp_msg`?